problems

```diff
@@ -0,0 +1,66 @@
+<?php
+//problema1 
+function sumarray($arr){
+    $sum=0;
+	for($i=0;$i<count($arr);$i++)
+	{
+		$sum=$sum+$arr[$i];
```

Comment:
Given an integer array of size N, compute the sum of all _even_ numbers in this array.

even inseamna pare